vsts/scripts/tagBuildImagesForRelease.sh

```diff
@@ -43,3 +43,4 @@ tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build:Oryx-CI.$RELEASE_TAG_NAME
 tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build:lts-versions-Oryx-CI.$RELEASE_TAG_NAME" "lts-versions" "lts-versions-$RELEASE_TAG_NAME"
 tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build:azfunc-jamstack-Oryx-CI.$RELEASE_TAG_NAME" "azfunc-jamstack" "azfunc-jamstack-$RELEASE_TAG_NAME"
 tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build:github-actions-Oryx-CI.$RELEASE_TAG_NAME" "github-actions" "github-actions-$RELEASE_TAG_NAME"
+tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build:vso-Oryx-CI.$RELEASE_TAG_NAME" "vso" "vso-$RELEASE_TAG_NAME"
```

Comment:
I missed adding this line because of which `vso` image is not being pushed to staging registry currently

Reply:
hmm curious .. then how did the vso teams get images from us?

Reply:
Until now they pick up the full build image `oryx/build`...this vso image is new and is also based on full build image but has a few tweaks on how `benv` would work with paths.